index.php

```diff
@@ -46,6 +46,7 @@
                     echo \Mamoi\ViewHelpers\IngredientViewHelper::displayAllIngredients($allIngredients);
                     ?>
                 </div>
+                <input class="submit" type="submit">
```

Comment:
I would suggest putting the attribute value="Search" so that the button displays Search rather than Submit.